rfd/0221-mcp-gateway.md

```diff
@@ -0,0 +1,129 @@
+---
+authors: Gabriel Corado (gabriel.oliveira@goteleport.com)
+state: draft
+---
+
+# RFD 0221 - MCP Gateway
+
+## Required Approvers
+
+* Engineering: @r0mant && @greedy52
+* Product: @klizhentas
+
+## What
```

Comment:
Would this design be extendable to a Remote Gateway configuration? From reading this, it seems we are creating a localized MCP Router, which would be great if we could also have it operate remotely.